### ISSUE TITLE
Breaks on node@0.10.26

### ISSUE THREAD

**Issue by Raynos** [Voxer/sse4_crc32#35]
You've changed to nan@2 in a minor version.

This breaks tchannel on node 0.10.26

```
In file included from ../src/sse4_crc32.cpp:18:
../node_modules/nan/nan.h:1178:39: error: no member named 'REPLACE_INVALID_UTF8' in 'v8::String'
                          v8::String::REPLACE_INVALID_UTF8;
                          ~~~~~~~~~~~~^
1 error generated.
make: *** [Release/obj.target/sse4_crc32/src/sse4_crc32.o] Error 1
```

**Comment by anandsuresh** [comment]
Fixed in release v3.4.0

**Comment by Raynos** [comment]
Thank you.
